packages/google_sign_in/ios/Classes/GoogleSignInPlugin.m

```diff
@@ -84,11 +83,15 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     }
   } else if ([call.method isEqualToString:@"signInSilently"]) {
     if ([self setAccountRequest:result]) {
-      [[GIDSignIn sharedInstance] signInSilently];
+      [[GIDSignIn sharedInstance] restorePreviousSignIn];
     }
   } else if ([call.method isEqualToString:@"isSignedIn"]) {
-    result(@([[GIDSignIn sharedInstance] hasAuthInKeychain]));
+    result(@([[GIDSignIn sharedInstance] hasPreviousSignIn]));
   } else if ([call.method isEqualToString:@"signIn"]) {
+    // TODO(jackson): It might be more appropriate to use controller of the FlutterView
+    UIViewController *controller = [UIApplication sharedApplication].keyWindow.rootViewController;
```

Comment:
@matthew-carroll do you have any concerns about using this approach to obtain a ViewController reference?

Reply:
I've updated it to use `[registrar messenger]`.